ios/StripeTerminalReactNative.swift

```diff
@@ -761,16 +822,21 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
 
     @objc(readReusableCard:resolver:rejecter:)
     func readReusableCard(params: NSDictionary, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
```

Comment:
we should remove this.

if we did want to implicitly use setup intents we'd need to do a lot more than just createSetupIntent.